etc/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE module PUBLIC
+  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+  "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+  <property name="charset" value="UTF-8"/>
+  <property name="severity" value="error"/>
+  <property name="fileExtensions" value="java, properties, xml"/>
+  <!-- Excludes all 'module-info.java' files              -->
+  <!-- See https://checkstyle.org/config_filefilters.html -->
+  <module name="BeforeExecutionExclusionFileFilter">
+    <property name="fileNamePattern" value="module\-info\.java$"/>
+  </module>
+  <!-- https://checkstyle.org/config_filters.html#SuppressionFilter -->
+  <module name="SuppressionFilter">
+    <property name="file" value="${basedir}/checkstyle/checkstyle-suppressions.xml"
```

Comment:
Assuming `${basedir}` is the repository root, then `${basedir}/.checkstyle/suppressions.xml`

Reply:
oh oops, I got this wrong, basedir is `<project root>/etc/checkstyle`, where the `checkstyle.xml` currentl is. Want to go for both the `checkstyle.xml` and `suppressions.xml` in `<project root>/.checkstyle`?

Reply:
both in there works good for me